### PR TITLE
Liuba- different bugs, including : meetid not found, /menu deselect cards etc.

### PR DESCRIPTION
**Pull request by Liubzzz**

### CODE REVIEW

File: src/components/TabletApp/LeftSide/UpcomingCards/UpcomingCards.tsx [hunk 1]
```diff
@@ -5,6 +5,7 @@ import SIZE from "../../../../constants/CustomSize";
 import { useNavigate } from "react-router";
 import COLORS from "../../../../constants/CustomColors";
 import { StringDecoder } from "string_decoder";
+import { useLocation } from "react-router-dom";
```
Comment (CatalinTiu) [inline src/components/TabletApp/LeftSide/UpcomingCards/UpcomingCards.tsx:8]: Maybe rearrange the imports,  the react-routers together, and the COLORS import separate from the rest as it's a custom import

File: src/components/TabletApp/LeftSide/UpcomingCards/UpcomingCards.tsx [hunk 2]
```diff
@@ -34,6 +35,7 @@ const UpcomingCards = ({
 
     const [joiners, setJoiners] = useState<(string | undefined)[]>();
     const [surplus, setSurplus] = useState<number>();
+    const location = useLocation();
```
Comment (CatalinTiu) [inline src/components/TabletApp/LeftSide/UpcomingCards/UpcomingCards.tsx:38]: location  is a bit vague for const name maybe?

**Review by CatalinTiu — APPROVED** [review]

File: src/components/TabletApp/LeftSide/UpcomingCards/UpcomingCards.tsx (hunk 2 above)
Comment (robert-szalkai) [reply src/components/TabletApp/LeftSide/UpcomingCards/UpcomingCards.tsx:38]: the constant name is ok. It is how on the official docs is used also

Comment (daniel-mita) [reply src/components/TabletApp/LeftSide/UpcomingCards/UpcomingCards.tsx:38]: This is fine as it is here

**Review by daniel-mita — APPROVED** [review]

**Review by robert-szalkai — APPROVED** [review]

**Pull request merged**